perfkitbenchmarker/pkb.py

```diff
@@ -331,23 +332,17 @@ def _LogCommandLineFlags():
   logging.info('Flag values:\n%s', '\n'.join(result))
 
 
-def RunBenchmarks(publish=True):
-  """Runs all benchmarks in PerfKitBenchmarker.
-
-  Args:
-    publish: A boolean indicating whether results should be published.
+def SetUpPKB():
+  """Set globals and environment variables for PKB.
 
-  Returns:
-    Exit status for the process.
+  Having this function separate from functions that do any work lets
+  us call this to set up integration tests.
```

Comment:
Changed the description to be more clear. I mentioned what it does, but I think the most important part is the postcondition - after running `SetUpPKB()`, you should be able to run any other PKB function and get useful results.